docs/src/spec/items/contracts.md

```diff
@@ -46,6 +46,22 @@ contract GuestBook {
 }
 ```
 
+Contract initialization may be configured using the `__init__` function.
```

Comment:
haha, started including that as part of a bigger docs revamp yesterday. But happy to take yours. Maybe add that the `__init__` function can only be called at *contract deployment time*  and can not be called at *runtime* 

https://github.com/ethereum/fe/pull/854/files

Reply:
your wording looks good. let me just close this PR  :laughing:

Reply:
As you like, turned it into a standalone PR as the rest is still a local WIP. Feel free to merge: 

https://github.com/ethereum/fe/pull/854